Expo1900/Expo1900Tests/ModelTest.swift

```diff
@@ -4,18 +4,41 @@ import XCTest
 class ModelTest: XCTestCase {
     
     func test_Exposition_decode호출시_올바른값을가져오는지() {
-        guard let result = Exposition.decode(with: "exposition_universelle_1900") else {
-            XCTFail("decode 실패함")
-            return }
-
-        XCTAssertEqual(result.title, "파리 만국박람회 1900(L'Exposition de Paris 1900)")
+        let result = Exposition.decode(with: "exposition_universelle_1900")
+        var title: String?
+            switch result {
+            case .success(let data):
+                title = data.title
+            default: break
+            }
```

Comment:
들여쓰기가 조금 어색한 것 같아요..!
추가로 switch 로 분기하는 것과 큰 차이는 없지만.. `Result` 타입에서 성공했을 때의 값만 필요하다면, `Result` 타입의 `get` 메서드도 활용해볼 수 있습니다.

Reply:
최초에는 또치 말씀대로 get메서드를 이용해 가져오긴 했는데 추후 해당 메서드는 이런 지금 작성한 방식으로 사용할 것 이라고 생각해 테스트 코드도 동일한 방식으로 작성하였습니다!

아울러, 들여쓰기가 어색하다고 말씀하신 부분은
```swift
            default:
                break }
```
이런 식으로 수정하면 괜찮을까요?

Reply:
엇 아뇨아뇨!
```suggestion
        switch result {
        case .success(let data):
            title = data.title
        default: break
        }
```
요렇게 바로 윗 라인에 선언된 `var title: String?`와 들여쓰기가 맞춰져야하는 것 아닌가..? 하는 생각이었어요!

Reply:
해당 부분은 테스트 케이스를 수정하면서 삭제되었습니다